### ISSUE TITLE
Track user's activity inside our extensions

### ISSUE THREAD

**Issue by andergmartins** [OSTraining/OSSystem#2]
Ask permission for that, in a popover. We can use it to track the user's activity and see what are the most used features for each extension. If required, we can create a new plugin for that.

The statistics are anonymous.

Check if it will require too much work.

**Comment by stevejburge** [comment]
Maybe not activity, so much as their setup?

What version of Joomla? What version of PHP? etc

**Comment by andergmartins** [comment]
Good point. Maybe what extensions are installed, how much installations are for each extension, per version...
